<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="reference_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #cccccc;</string>
    </property>
    <property name="text">
     <string>기준 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camera_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #cccccc;</string>
    </property>
    <property name="text">
     <string>현재 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ref_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>760</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>기준 이미지 설정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roi_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>760</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI 선택</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_stop_button">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>760</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>660</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
          QLabel {
              font-size: 24px;
              font-weight: bold;
              padding: 10px;
              border-radius: 5px;
              background-color: #f0f0f0;
          }</string>
    </property>
    <property name="text">
     <string>검사결과</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="threshold_slider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>725</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="threshold_label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>720</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>임계값: 100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="camera_comboBox">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>790</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text_label1">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>757</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>카메라 선택</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
